v6_carrier_py/master.py

```diff
@@ -131,15 +132,15 @@ def fit_pipeline(client: ClientContainerProtocol, data, pipe: Pipeline, features
                         `predict` method.
     :param features: The features that should be used in the fitting of the pipeline.
     :param target: The field that should be used as target for the machine learning algorithm.
-    :param identifying_columns: The identifying fields for joining datasets.
+    :param keys: The identifying fields for joining datasets.
```

Comment:
I agree `identifying_columns` is a bit verbose, but `keys` is so much used in other contexts. Maybe we can name it `on` (like in pandas) or `join_keys`, `on_keys`.

Reply:
I renamed it to `merge_keys`. In the final product this is going to be gone anyway.